packages/eslint-plugin/docs/rules/no-floating-promises.mdx

```diff
@@ -134,6 +134,12 @@ await createMyThenable();
 This option, which is `true` by default, allows you to stop the rule reporting promises consumed with void operator.
 This can be a good way to explicitly mark a promise as intentionally not awaited.
 
+:::warning
+Voiding a Promise doesn't handle it, but just ignores the outcome. Such Promise rejection will still lead to an
+[`unhandledrejection` event](https://developer.mozilla.org/en-US/docs/Web/API/Window/unhandledrejection_event)
+being fired and error logs being printed. Refer to the list at the top of the page for the possible ways to handle a promise.
+:::
```

Comment:
(praise) I quite like that you've used the `:::warning` block here 🙂 

... but I feel like the wording wanders a bit off topic. What do you think of something like this?

```suggestion
:::warning
Voiding a promise doesn't handle it or change its behavior in any way. It is equivalent to an `// eslint-disable-next-line @typescript-eslint/no-floating-promises` comment.
:::
```

Reply:
Quoting the `disable-next-line` fact is great and I can add it. However, my idea was to exactly represent what happend to me and what I described in the issue (the code example).

So, I think it is important to include a link to mdn page about `unhandledrejection`, which can explain surely better what is it.

About the comment above, I think could be interesting to give a hint about how can a person handle a promise.